scylla-macros/src/serialize/row.rs

```diff
@@ -1,3 +1,6 @@
+// See: https://github.com/TedDriggs/darling/issues/293
+#![allow(clippy::manual_unwrap_or_default)]
```

Comment:
We need to remove those after the issue is fixed in darling. Please open an issue about it so we don't forget

Reply:
Also do we need it on the whole file? Isn't it possible to only have it in problematic places?

Reply:
> Also do we need it on the whole file? Isn't it possible to only have it in problematic places?

I tried putting it in the following places:
- next to the fields marked with `#[darling(default)]` attribute (tried both after and before darling attribute)
- next to the struct with marked fields

None of these worked. Do you have any more ideas?

Reply:
You can also try putting the `allow` before or after the `#[derive(FromAttributes)]` part.

Reply:
> You can also try putting the `allow` before or after the `#[derive(FromAttributes)]` part.

Unfortunately, this does not work